app/presenters/two_factor_options_presenter.rb

```diff
@@ -108,7 +108,7 @@ def webauthn_option
   end
 
   def webauthn_platform_option
-    return [] if piv_cac_required? || !IdentityConfig.store.platform_auth_set_up_enabled
```

Comment:
Interesting logic I wasn't aware of... seems like this might have prevented people from seeing F/T as an option for signing in when the feature was disabled? That sounds like it wasn't working correctly 😬